src/test/java/net/sf/jsqlparser/statement/select/SelectTest.java

```diff
@@ -52,6 +89,13 @@ public void testMultiPartTableNameWithServerNameAndDatabaseNameAndSchemaName() t
         final String statement = "SELECT columnName FROM [server-name\\server-instance].databaseName.schemaName.tableName";
         assertSqlCanBeParsedAndDeparsed(statement, false,
                 parser -> parser.withSquareBracketQuotation(true));
+        assertDeparse(new Select().withSelectBody(new PlainSelect()
```

Comment:
Don't change existing tests in this way. Create new tests.

Reply:
see my comment above at AlterTest.java